### ISSUE TITLE
Support fro Nexus v3 broken

### ISSUE THREAD

**Issue by uvwildos** [dependency-check/DependencyCheck#5319]
**Describe the bug**
Due to the issues with the central analyzer I try to use our local nexusv3 pro providing a  proxy for central.
However, I cannot get it to work. 

**Version of dependency-check used**
cli based on CLI version 7.4.4

**Log file**
https://github.com/jeremylong/DependencyCheck/issues/5298

**To Reproduce**
https://github.com/jeremylong/DependencyCheck/issues/5298

show the cli parameters and how they work in curl

**Expected behavior**
some more detailed feedback why it fails

**Additional context**
i found a note that nexus v3 support is not working. so please clarify.

**Comment by jeremylong** [comment]
I do not believe any of the core maintainers have access to a Nexus V3 server. The integrations were provided by other contributors via PR. If you are facing an issue and can help resolve any errors you encounter - PRs are welcome.

**Comment by uvwildos** [comment]
i keep it in mind and I wish I have a spare moment some time.
However, I would kindly ask for a note in the docs to keep expectations low |
since the docs kind of say it should work !?!
